sphinx/tutorial/rose/suites.rst

```diff
@@ -32,6 +49,8 @@ A Rose suite configuration is a Cylc :term:`run directory` containing a
 
    :rose:conf:`rose-suite.conf[env]`
       Environment variables for use by the whole suite.
+   :rose:conf:`rose-suite.conf[template variables]`
+      Generic variables for use in the ``flow.cylc`` file.
```

Comment:
I would be tempted to remove the separate Jinja2 / EmPy sections and revise the note to explain that there used to be separate sections for them. (want new people to use the best practice):

```rst
   :rose:conf:`rose-suite.conf[template variables]`
      Generic variables for use by Jinja2 / EmPy in the ``flow.cylc`` file.
```